samples/ios/app/glean-sample-app/ViewController.swift

```diff
@@ -56,6 +57,10 @@ class ViewController: UIViewController {
         // Increment the custom counter that goes into the sample ping
         Custom.counter.add()
         Pings.shared.sample.submit()
+
+        // Activate the glean-sample-framework library function that triggers Glean metric
+        // recording
+        GleanSampleFramework.doTheThing(withIntensity: 9002)
```

Comment:
![OVER 9000](https://media.giphy.com/media/ejfQbLsh2rFfi/giphy.gif)

Reply:
I saw this and borrowed it :)